spec/requests/admin/roles/permissions_spec.rb

```diff
@@ -21,7 +21,7 @@
 
     it "shows the correct permissions" do
       subject
-      expect(response.body).to include("application.passported._")
+      expect(response.body).not_to include("application.passported._")
```

Comment:
Does this test make sense still now that passported permissions have been removed?

Reply:
👍 have removed this test